doc/flame/components.md

```diff
@@ -154,6 +154,48 @@ void update(double dt) {
 ```
 
 
+### Blueprints
+
+Flame also offers a virtual approach for grouping components which allows the developer to keep
+their components grouped "code wise", but the components (or other things, like a
+`ContactCallback` for Forge2d Games) are individually added to the game and there isn't a parent
+component.
```

Comment:
avoid some repetition:
```suggestion
Flame also offers a virtual approach for grouping components that allows the developer to keep them together "code wise", but each component (or other things, like a `ContactCallback` for Forge2d Games) are individually added to the game without a parent.
```

Reply:
arent' `ContactCallback` components?? your impl seems to take only Components and nothing else

Reply:
> arent' ContactCallback components?? your impl seems to take only Components and nothing else
Look on the `Forge2dBlueprint` implementation 😉